ingest/ingestHelpers.py

```diff
@@ -141,119 +151,142 @@ def get_distributions(ds):
     return distributions
 # end
 
+#############################################
+#    Implementation of Main(...):
+#
 
+def Main(get_objects_query_location, describe_object_query_location,
```

Comment:
don't start methods with capital letters - that is reserved in python for class definitions.